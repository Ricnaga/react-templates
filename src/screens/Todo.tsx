export function TodoScreen() {
  return <h1>This is todo</h1>;
}
