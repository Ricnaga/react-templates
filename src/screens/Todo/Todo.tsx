import { Grid, GridItem } from '@chakra-ui/react';
import { useState } from 'react';
import {
  CardTodo,
  HandleChangeProps,
  HandleMoveProps,
  TodoData,
  TodoStatus,
} from './components/CardTodo';

const MOCK_DATA: Array<TodoData> = [
  { id: Math.random().toString(), status: 1, description: 'Pegar casaco' },
  { id: Math.random().toString(), status: 2, description: 'Pegar calÃ§a' },
  { id: Math.random().toString(), status: 3, description: 'Pegar sapato' },
  { id: Math.random().toString(), status: 1, description: 'Pegar blusa' },
  { id: Math.random().toString(), status: 2, description: 'Pegar bolsa' },
];

const filterByStatus = (status: TodoStatus) =>
  MOCK_DATA.filter((data) => data.status === status);

const mapById = (id: string, description: string, data: Array<TodoData>) =>
  data.map((newState) =>
    newState.id === id ? { ...newState, description } : newState,
  );

const filterById = (id: string, data: Array<TodoData>) =>
  data.filter((newState) => newState.id !== id);

export function TodoScreen() {
  const [todo, setTodo] = useState<Array<TodoData>>(
    filterByStatus(TodoStatus.TODO),
  );
  const [doing, setDoing] = useState<Array<TodoData>>(
    filterByStatus(TodoStatus.DOING),
  );
  const [done, setDone] = useState<Array<TodoData>>(
    filterByStatus(TodoStatus.DONE),
  );

  const handleChangeStatus = ({
    id,
    description,
    status,
  }: HandleChangeProps) => {
    if (status === TodoStatus.DOING) {
      setDoing((state) => mapById(id, description, state));
      return;
    }

    setTodo((state) => mapById(id, description, state));
  };

  const handleRemoveTodo = (id: string, status: TodoStatus) => {
    if (status === TodoStatus.DOING) {
      setDoing((state) => filterById(id, state));
      return;
    }

    setTodo((state) => filterById(id, state));
  };

  const handleAddTodo = (data: TodoData) => {
    if (data.status === TodoStatus.DOING) {
      setDoing([...doing, data]);
      return;
    }

    setTodo([...todo, data]);
  };

  const handleMoveTodo = ({ data, moveTo }: HandleMoveProps) => {
    switch (data.status) {
      case TodoStatus.DOING: {
        setDoing((state) => filterById(data.id, state));

        if (moveTo === 'NEXT') {
          setDone((state) => [...state, { ...data, status: TodoStatus.DONE }]);
          return;
        }

        setTodo((state) => [...state, { ...data, status: TodoStatus.TODO }]);
        return;
      }
      case TodoStatus.DONE: {
        setDone((state) => filterById(data.id, state));

        setDoing((state) => [...state, { ...data, status: TodoStatus.DOING }]);
        return;
      }
      default: {
        setTodo((state) => filterById(data.id, state));
        setDoing((state) => [...state, { ...data, status: TodoStatus.DOING }]);
      }
    }
  };

  const cardComponent = [
    { title: TodoStatus.TODO, data: todo },
    { title: TodoStatus.DOING, data: doing },
    { title: TodoStatus.DONE, data: done },
  ];

  return (
    <Grid gap={4} templateColumns="repeat(3,1fr)">
      {cardComponent.map(({ data, title }) => (
        <GridItem key={title}>
          <CardTodo
            title={title}
            data={data}
            handleChange={handleChangeStatus}
            handleRemove={handleRemoveTodo}
            handleAdd={handleAddTodo}
            handleMove={handleMoveTodo}
          />
        </GridItem>
      ))}
    </Grid>
  );
}
