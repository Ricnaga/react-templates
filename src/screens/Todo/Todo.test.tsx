import { render } from '@application/test/testing-library';

import { TodoScreen } from './Todo';

describe('Page: HomePage', () => {
  it('should render correctly', () => {
    const { container } = render(<TodoScreen />);
    expect(container).toMatchSnapshot();
  });
});
