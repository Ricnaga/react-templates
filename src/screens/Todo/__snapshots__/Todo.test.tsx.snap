// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: HomePage should render correctly 1`] = `
<div>
  <div
    class="css-1t9exsr"
  >
    <div
      class="css-0"
    >
      <div
        class="chakra-card css-jv498s"
      >
        <div
          class="chakra-card__header css-svdwki"
        >
          <p
            class="chakra-text css-7otdp4"
          >
            To Do
          </p>
        </div>
        <div
          class="chakra-card__body css-1idwstw"
        >
          <div
            class="css-1ym6jsm"
          >
            <div
              class="css-1pip4vl"
            >
              <p
                class="chakra-text css-w6oofq"
              >
                Pegar casaco
              </p>
            </div>
            <div
              class="css-1ls3ug2"
            >
              <button
                aria-label="Edit icon"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M11 4H4a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-7"
                    />
                    <path
                      d="M18.5 2.5a2.121 2.121 0 0 1 3 3L12 15l-4 1 1-4 9.5-9.5z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Remove icon"
                class="chakra-button css-1974nvi"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M19.452 7.5H4.547a.5.5 0 00-.5.545l1.287 14.136A2 2 0 007.326 24h9.347a2 2 0 001.992-1.819L19.95 8.045a.5.5 0 00-.129-.382.5.5 0 00-.369-.163zm-9.2 13a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zm5 0a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zM22 4h-4.75a.25.25 0 01-.25-.25V2.5A2.5 2.5 0 0014.5 0h-5A2.5 2.5 0 007 2.5v1.25a.25.25 0 01-.25.25H2a1 1 0 000 2h20a1 1 0 000-2zM9 3.75V2.5a.5.5 0 01.5-.5h5a.5.5 0 01.5.5v1.25a.25.25 0 01-.25.25h-5.5A.25.25 0 019 3.75z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Next Task"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M13.584,12a2.643,2.643,0,0,1-.775,1.875L3.268,23.416a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L.768,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,13.584,12Z"
                    />
                    <path
                      d="M23.75,12a2.643,2.643,0,0,1-.775,1.875l-9.541,9.541a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L10.934,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,23.75,12Z"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
          <div
            class="css-1ym6jsm"
          >
            <div
              class="css-1pip4vl"
            >
              <p
                class="chakra-text css-w6oofq"
              >
                Pegar blusa
              </p>
            </div>
            <div
              class="css-1ls3ug2"
            >
              <button
                aria-label="Edit icon"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M11 4H4a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-7"
                    />
                    <path
                      d="M18.5 2.5a2.121 2.121 0 0 1 3 3L12 15l-4 1 1-4 9.5-9.5z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Remove icon"
                class="chakra-button css-1974nvi"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M19.452 7.5H4.547a.5.5 0 00-.5.545l1.287 14.136A2 2 0 007.326 24h9.347a2 2 0 001.992-1.819L19.95 8.045a.5.5 0 00-.129-.382.5.5 0 00-.369-.163zm-9.2 13a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zm5 0a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zM22 4h-4.75a.25.25 0 01-.25-.25V2.5A2.5 2.5 0 0014.5 0h-5A2.5 2.5 0 007 2.5v1.25a.25.25 0 01-.25.25H2a1 1 0 000 2h20a1 1 0 000-2zM9 3.75V2.5a.5.5 0 01.5-.5h5a.5.5 0 01.5.5v1.25a.25.25 0 01-.25.25h-5.5A.25.25 0 019 3.75z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Next Task"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M13.584,12a2.643,2.643,0,0,1-.775,1.875L3.268,23.416a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L.768,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,13.584,12Z"
                    />
                    <path
                      d="M23.75,12a2.643,2.643,0,0,1-.775,1.875l-9.541,9.541a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L10.934,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,23.75,12Z"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="chakra-card__footer css-1a7q9t8"
        >
          <button
            aria-label="Add icon"
            class="chakra-button css-tuydmg"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="chakra-icon css-onkibi"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                fill="currentColor"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="chakra-card css-jv498s"
      >
        <div
          class="chakra-card__header css-clqsa8"
        >
          <p
            class="chakra-text css-7otdp4"
          >
            Doing
          </p>
        </div>
        <div
          class="chakra-card__body css-1idwstw"
        >
          <div
            class="css-1ym6jsm"
          >
            <div
              class="css-0"
            >
              <button
                aria-label="Previous task"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M10.416,12a2.643,2.643,0,0,1,.775-1.875L20.732.584a1.768,1.768,0,0,1,2.5,2.5l-8.739,8.739a.25.25,0,0,0,0,.354l8.739,8.739a1.768,1.768,0,0,1-2.5,2.5l-9.541-9.541A2.643,2.643,0,0,1,10.416,12Z"
                    />
                    <path
                      d="M.25,12a2.643,2.643,0,0,1,.775-1.875L10.566.584a1.768,1.768,0,0,1,2.5,2.5L4.327,11.823a.25.25,0,0,0,0,.354l8.739,8.739a1.768,1.768,0,0,1-2.5,2.5L1.025,13.875A2.643,2.643,0,0,1,.25,12Z"
                    />
                  </g>
                </svg>
              </button>
            </div>
            <div
              class="css-rklm6r"
            >
              <p
                class="chakra-text css-w6oofq"
              >
                Pegar cal√ßa
              </p>
            </div>
            <div
              class="css-1ls3ug2"
            >
              <button
                aria-label="Edit icon"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M11 4H4a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-7"
                    />
                    <path
                      d="M18.5 2.5a2.121 2.121 0 0 1 3 3L12 15l-4 1 1-4 9.5-9.5z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Remove icon"
                class="chakra-button css-1974nvi"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M19.452 7.5H4.547a.5.5 0 00-.5.545l1.287 14.136A2 2 0 007.326 24h9.347a2 2 0 001.992-1.819L19.95 8.045a.5.5 0 00-.129-.382.5.5 0 00-.369-.163zm-9.2 13a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zm5 0a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zM22 4h-4.75a.25.25 0 01-.25-.25V2.5A2.5 2.5 0 0014.5 0h-5A2.5 2.5 0 007 2.5v1.25a.25.25 0 01-.25.25H2a1 1 0 000 2h20a1 1 0 000-2zM9 3.75V2.5a.5.5 0 01.5-.5h5a.5.5 0 01.5.5v1.25a.25.25 0 01-.25.25h-5.5A.25.25 0 019 3.75z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Next Task"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M13.584,12a2.643,2.643,0,0,1-.775,1.875L3.268,23.416a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L.768,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,13.584,12Z"
                    />
                    <path
                      d="M23.75,12a2.643,2.643,0,0,1-.775,1.875l-9.541,9.541a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L10.934,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,23.75,12Z"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
          <div
            class="css-1ym6jsm"
          >
            <div
              class="css-0"
            >
              <button
                aria-label="Previous task"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M10.416,12a2.643,2.643,0,0,1,.775-1.875L20.732.584a1.768,1.768,0,0,1,2.5,2.5l-8.739,8.739a.25.25,0,0,0,0,.354l8.739,8.739a1.768,1.768,0,0,1-2.5,2.5l-9.541-9.541A2.643,2.643,0,0,1,10.416,12Z"
                    />
                    <path
                      d="M.25,12a2.643,2.643,0,0,1,.775-1.875L10.566.584a1.768,1.768,0,0,1,2.5,2.5L4.327,11.823a.25.25,0,0,0,0,.354l8.739,8.739a1.768,1.768,0,0,1-2.5,2.5L1.025,13.875A2.643,2.643,0,0,1,.25,12Z"
                    />
                  </g>
                </svg>
              </button>
            </div>
            <div
              class="css-rklm6r"
            >
              <p
                class="chakra-text css-w6oofq"
              >
                Pegar bolsa
              </p>
            </div>
            <div
              class="css-1ls3ug2"
            >
              <button
                aria-label="Edit icon"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M11 4H4a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-7"
                    />
                    <path
                      d="M18.5 2.5a2.121 2.121 0 0 1 3 3L12 15l-4 1 1-4 9.5-9.5z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Remove icon"
                class="chakra-button css-1974nvi"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M19.452 7.5H4.547a.5.5 0 00-.5.545l1.287 14.136A2 2 0 007.326 24h9.347a2 2 0 001.992-1.819L19.95 8.045a.5.5 0 00-.129-.382.5.5 0 00-.369-.163zm-9.2 13a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zm5 0a.75.75 0 01-1.5 0v-9a.75.75 0 011.5 0zM22 4h-4.75a.25.25 0 01-.25-.25V2.5A2.5 2.5 0 0014.5 0h-5A2.5 2.5 0 007 2.5v1.25a.25.25 0 01-.25.25H2a1 1 0 000 2h20a1 1 0 000-2zM9 3.75V2.5a.5.5 0 01.5-.5h5a.5.5 0 01.5.5v1.25a.25.25 0 01-.25.25h-5.5A.25.25 0 019 3.75z"
                    />
                  </g>
                </svg>
              </button>
              <button
                aria-label="Next Task"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M13.584,12a2.643,2.643,0,0,1-.775,1.875L3.268,23.416a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L.768,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,13.584,12Z"
                    />
                    <path
                      d="M23.75,12a2.643,2.643,0,0,1-.775,1.875l-9.541,9.541a1.768,1.768,0,0,1-2.5-2.5l8.739-8.739a.25.25,0,0,0,0-.354L10.934,3.084a1.768,1.768,0,0,1,2.5-2.5l9.541,9.541A2.643,2.643,0,0,1,23.75,12Z"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="chakra-card__footer css-1a7q9t8"
        >
          <button
            aria-label="Add icon"
            class="chakra-button css-tuydmg"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="chakra-icon css-onkibi"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                fill="currentColor"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="chakra-card css-jv498s"
      >
        <div
          class="chakra-card__header css-zsygm5"
        >
          <p
            class="chakra-text css-7otdp4"
          >
            Done
          </p>
        </div>
        <div
          class="chakra-card__body css-1idwstw"
        >
          <div
            class="css-1ym6jsm"
          >
            <div
              class="css-0"
            >
              <button
                aria-label="Previous task"
                class="chakra-button css-1nn3802"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="currentColor"
                  >
                    <path
                      d="M10.416,12a2.643,2.643,0,0,1,.775-1.875L20.732.584a1.768,1.768,0,0,1,2.5,2.5l-8.739,8.739a.25.25,0,0,0,0,.354l8.739,8.739a1.768,1.768,0,0,1-2.5,2.5l-9.541-9.541A2.643,2.643,0,0,1,10.416,12Z"
                    />
                    <path
                      d="M.25,12a2.643,2.643,0,0,1,.775-1.875L10.566.584a1.768,1.768,0,0,1,2.5,2.5L4.327,11.823a.25.25,0,0,0,0,.354l8.739,8.739a1.768,1.768,0,0,1-2.5,2.5L1.025,13.875A2.643,2.643,0,0,1,.25,12Z"
                    />
                  </g>
                </svg>
              </button>
            </div>
            <div
              class="css-1pip4vl"
            >
              <p
                class="chakra-text css-w6oofq"
              >
                Pegar sapato
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
